### PR TITLE
Update README.md

### LINKED ISSUE #10: Latest 4 solc versions are missed.
When listing the remote versions the loop misses the last 4 versions in the list.

```js
for (var i = sources.length - 1; i >= 0; i--) {
    // FIXME: use build as well
    var version = sources[i].match(/^soljson-v([0-9.]*)-.*.js$/);

    // Skip invalid lines
    if (!version) {
      continue;
    }

    if (version[1] === wanted) {
      downloadBinary(sources[i].match(/^soljson-(.*).js$/)[1]);
      return;
    }
  }
```

### PR DESCRIPTION
**Pull request by Denton-L**
Fixes #10, @axic.

### CODE REVIEW

**Comment by axic** [comment]
`loadRemoteVersion` is asynchronous so it is rather confusing:

```
	var solc = require('solc');

	// getting the development snapshot
	var solcSnapshot = solc.loadRemoteVersion('latest', function(err, solcSnapshot) {
            if (err)
                // An error was encountered, display and quit

            var output = solcSnapshot.compile( "contract t { function g() {} }", 1 );
        });
```

**Comment by axic** [comment]
Also please check #3 for the other changes to the README.

**Comment by Denton-L** [comment]
Alright, I've updated the PR to include the corrected example code and also the additions from #3.

File: README.md [hunk 1]
```diff
@@ -1,61 +1,92 @@
...
-	var output = solcV011.compile( "contract t { function g() {} }", 1 );
+### Using the Latest Development Snapshot
+
+By default, the npm version is only created for releases. This prevents people from deploying contracts with non-release versions because they are less stable and harder to verify. If you would like to use the latest development snapshot (at your own risk!), you may use the following example code.
+
+```javascript
+var solc = require('solc');
+
+// getting the development snapshot
+var solcSnapshot = solc.loadRemoteVersion('latest', function(err, solcSnapshot) {
```
Comment (axic) [inline README.md]: `var solcSnapshot = ` is not needed, please remove (sorry for having it my example)

File: README.md [hunk 2]
```diff
@@ -1,61 +1,92 @@
...
+```
 
-In order to allow compiling contracts using a specific version of solidity, the `solc.useVersion` method is available. This returns a new solc object using the version provided. **Note**: version strings must match the version substring of the files availble in `/bin/soljson-*.js`. See below for an example.
+If the version is not available locally, you can use `solc.loadRemoteVersion(version, callback)` to load it directly from GitHub.
 
-	var solc = require('solc');
-	// by default the latest version is used
-	// ie: solc.useVersion('latest')
+You can also load the "binary" manually and use `setupMethods` to create the familiar wrapper functions described above:
+`var sol = solc.setupMethods(require("/my/local/soljson.js"))`.
```
Comment (axic) [inline README.md]: `var sol =` should be `var solc =`

**Comment by Denton-L** [comment]
Corrected both issues :)

**Comment by chriseth** [comment]
Looks good to me!

**Comment by axic** [comment]
👍

**Comment by Denton-L** [comment]
Sorry, I tacked on a few more minor commits.

**Pull request merged**